include/input.tcc

```diff
@@ -51,80 +53,70 @@ std::vector<std::shared_ptr<Point<Tdim>>> Input<Tdim>::read_vertices(
 
 template <unsigned Tdim>
 std::vector<std::shared_ptr<Point<Tdim>>> Input<Tdim>::read_elements(
-        const std::string& filename) {
-
-    //! Element Coordinate and id container
-    std::vector<std::shared_ptr<Point<Tdim>>> elements_;
```

Comment:
All these next 10 lines initialise variables, which are not in your code. These changes were undone. Do not undo others' changes, without sufficient explanation.